src/vs/workbench/contrib/testing/common/testingContentProvider.ts

```diff
@@ -61,12 +62,14 @@ export class TestingContentProvider implements IWorkbenchContribution, ITextMode
 				break;
 			}
 			case TestUriType.ResultMessage: {
-				const message = test.tasks[parsed.taskIndex].messages[parsed.messageIndex]?.message;
-				if (typeof message === 'string') {
-					text = message;
-				} else if (message) {
-					text = message.value;
-					language = this.languageService.createById('markdown');
+				const message = test.tasks[parsed.taskIndex].messages[parsed.messageIndex];
+				if (message) {
+					if (typeof message.message === 'string') {
+						text = message.type === TestMessageType.Info ? removeAnsiEscapeCodes(message.message) : message.message;
```

Comment:
Should the else condition in this also call `renderStringAsPlaintext`?

Reply:
no, since this is already a plaintext string in this branch (vs an `IMarkdownString`)